frontend/src/styles/ExpenseList.css

```diff
@@ -0,0 +1,14 @@
+.expenseform{
+    display: grid;
+    height: 100%;
```

Comment:
Height rule does not have any impact on the style, it should be targeted to the Form component. I suggest adding the styling under the "main" element, so you can there set the height of its direct child elements (ExpensePresentation and ExpenseList)

Reply:
Resolved in 82a9efe6dc53e25951eeecdd6b552f519d905b00